.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,13 @@
 ### Proposed behaviour
 
 <!--
-A clear and concise description of what changes this PR makes.
-
-If applicable, add screenshots of a codesandbox to help explain your request. You can paste these directly into GitHub.
-
-Please DO NOT share screenshots or the source code of your project.
-
-You can create a codesandbox to show the behaviour before/after this pull request by forking this template https://codesandbox.io/s/carbon-quickstart-j5pb2
-
-If you include a CodeSandbox link, the bot will fork it with the new built version of carbon.
-If you have a commit that includes fixes #XXXX and issue #XXXX has a CodeSandbox link in the body, the bot will fork
-it with the new built version of carbon.
+A clear and concise description of what changes this PR makes. If applicable, include any UI screenshots to help explain your request. If you are a Carbon consumer who is contributing, please DO NOT share any screenshots or source code of your own project.
 -->
 
 ### Current behaviour
 
 <!--
-A clear and concise description of the behaviour before this change.
-
-If applicable, add screenshots. You can paste these directly into GitHub.
+A clear and concise description of the behaviour before this change. If applicable, include any UI screenshots to help explain your request. If you are a Carbon consumer who is contributing, please DO NOT share any screenshots or source code of your own project.
```

Comment:
**suggestion**: also mention here that UI screenshots from products should not be included either